WolvenKit/Views/Dialogs/Windows/SelectFacialAnimationPathDialog.xaml

```diff
@@ -104,6 +106,32 @@
                         SelectedValue="{Binding SelectedGraph}">
                     </ComboBox>
 
+
+                    <Label Content="Animation entries" HorizontalAlignment="Left"
+                           Grid.Row="3"
+                           Grid.Column="0"
+                           Background="#252525" BorderThickness="0" /> <!-- Label for the dropdown -->
+
+                    <ComboBox
+                        x:Name="AnimEntryDropdown"
+                        Grid.Row="3"
```

Comment:
Can you write Grid.Row / Grid.Column at the top, or just after x:Name when present?